config/locales/de/proforma_errors.yml

```diff
@@ -0,0 +1,7 @@
+---
+de:
+  proforma_errors:
+    import: Es ist ein Fehler beim Importieren der ProformaXML ZIP-Datei aufgetreten.
+    import_default: "%{error}"
```

Comment:
What is the benefit of this locale (in contrast to passing in the `msg` directly to the `default`)?